transaction-status/src/lib.rs

```diff
@@ -556,6 +588,20 @@ impl ConfirmedBlock {
                 ),
             ),
             TransactionDetails::None => (None, None),
+            TransactionDetails::Accounts => (
+                Some(
+                    self.transactions
+                        .into_iter()
+                        .map(|tx_with_meta| {
+                            tx_with_meta.build_json_accounts(
+                                options.max_supported_transaction_version,
+                                options.show_rewards,
```

Comment:
It makes sense to pass this option through here but looks like we don't do that for the "full" details variant, could you  please make that consistent in a follow up PR?